front/projecttask.form.php

```diff
@@ -50,6 +50,9 @@
 if (!isset($_GET["projecttasks_id"])) {
    $_GET["projecttasks_id"] = "";
 }
+if (!isset($_GET["withtemplate"])) {
```

Comment:
as i see, this part is never used, when i test to add a task in my project template, the withtemplate GET var is not passed